exchanges/binance/binance.go

```diff
@@ -26,14 +26,14 @@ const (
 	apiURL = "https://api.binance.com"
 
 	// Public endpoints
-	exchangeInfo      = "/api/v1/exchangeInfo"
-	orderBookDepth    = "/api/v1/depth"
-	recentTrades      = "/api/v1/trades"
-	historicalTrades  = "/api/v1/historicalTrades"
-	aggregatedTrades  = "/api/v1/aggTrades"
-	candleStick       = "/api/v1/klines"
+	exchangeInfo      = "/api/v3/exchangeInfo"
+	orderBookDepth    = "/api/v3/depth"
+	recentTrades      = "/api/v3/trades"
+	historicalTrades  = "/api/v3/historicalTrades"
```

Comment:
We haven't gotten functionality for this, will make a note to expand and use the v3 endpoint. Unless you want to add it while you are here? If not all good let me know.

Reply:
Will keep that for a follow up change I think 👌

Reply:
Perfect 👍